src/components/articles/userArticles.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import "../../styles/app.scss";
+import moment from "moment";
+import { getArticlesAction } from "../../actions/getArticles";
+import { Link } from "react-router-dom";
+
+export class UserArticles extends Component {
+  constructor(props) {
+    super(props);
+  }
+  componentDidMount() {
+    this.props.getArticlesAction("?email=" + sessionStorage.getItem("email"));
+  }
+  render() {
+    const articles = this.props.articles;
+    const articleList =
+      articles && articles.length ? (
+        articles.map(article => {
+          return (
+            <div className="article" key={article.slug}>
+              <div className="inner">
+                <div className="article-card">
+                  <h3>{article.title.substring(0, 30)} ...</h3>
+                  <p>
+                    <b>{article.description.substring(0, 60)} ...</b>
+                  </p>
+                  <p>{article.body.substring(0, 60)} ...</p>
+                  <span>
+                    <b>Author: {article.author.username}</b>
+                  </span>
+                  <br />
+                  <span className="card-date">
+                    Created at{" "}
```

Comment:
@patrickf949 remove this

Reply:
thanks @anyatibrian, this is necessary to add a space between `Created at` and the time collected from the backend